applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -218,11 +220,12 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	size_t num_streams = ARRAY_SIZE(streams);
 	size_t data_size = size / num_streams;
 
-	if (streams[0].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
-		return -ECANCELED;
-	}
-
 	for (size_t i = 0U; i < num_streams; i++) {
+		if (streams[i].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+			LOG_DBG("Stream %d not in streaming state", i);
```

Comment:
I'm surprised this doesn't trigger an error in CI. Is the NCS CI more relaxed than the Zephyr one?

Reply:
I guess the NCS CI is more relaxed.
In the nrf5340_audio application we only use "for int i" and %d, except in the le_audio_(...) files, since these files are based on Zephyr samples.

We actually have a task for going through the "size_t i" and %d vs %u in all the logs in the application code.